src/Microsoft.VisualBasic/src/Microsoft/VisualBasic/Constants.vb

```diff
@@ -20,5 +20,36 @@ Namespace Global.Microsoft.VisualBasic
         'vbCompareMethod enum values
         Public Const vbBinaryCompare As CompareMethod = CompareMethod.Binary
         Public Const vbTextCompare As CompareMethod = CompareMethod.Text
+
+        'MsgBoxResult
+        Public Const vbAbort As MsgBoxResult = MsgBoxResult.Abort
+        Public Const vbCancel As MsgBoxResult = MsgBoxResult.Cancel
+        Public Const vbIgnore As MsgBoxResult = MsgBoxResult.Ignore
+        Public Const vbNo As MsgBoxResult = MsgBoxResult.No
+        Public Const vbOK As MsgBoxResult = MsgBoxResult.Ok
+        Public Const vbRetry As MsgBoxResult = MsgBoxResult.Retry
+        Public Const vbYes As MsgBoxResult = MsgBoxResult.Yes
+
+        'MsgBoxStyle
+        'You may BitOr one value from each group
+        Public Const vbAbortRetryIgnore As MsgBoxStyle = MsgBoxStyle.AbortRetryIgnore
+        Public Const vbApplicationModal As MsgBoxStyle = MsgBoxStyle.ApplicationModal
+        Public Const vbCritical As MsgBoxStyle = MsgBoxStyle.Critical
+        Public Const vbDefaultButton1 As MsgBoxStyle = MsgBoxStyle.DefaultButton1
+        Public Const vbDefaultButton2 As MsgBoxStyle = MsgBoxStyle.DefaultButton2
+        Public Const vbDefaultButton3 As MsgBoxStyle = MsgBoxStyle.DefaultButton3
+        Public Const vbExclamation As MsgBoxStyle = MsgBoxStyle.Exclamation
+        Public Const vbInformation As MsgBoxStyle = MsgBoxStyle.Information
+        Public Const vbMsgBoxHelp As MsgBoxStyle = MsgBoxStyle.MsgBoxHelp
+        Public Const vbMsgBoxRight As MsgBoxStyle = MsgBoxStyle.MsgBoxRight
+        Public Const vbMsgBoxRtlReading As MsgBoxStyle = MsgBoxStyle.MsgBoxRtlReading
+        Public Const vbMsgBoxSetForeground As MsgBoxStyle = MsgBoxStyle.MsgBoxSetForeground
+        Public Const vbOKCancel As MsgBoxStyle = MsgBoxStyle.OkCancel
+        Public Const vbOKOnly As MsgBoxStyle = MsgBoxStyle.OkOnly
+        Public Const vbQuestion As MsgBoxStyle = MsgBoxStyle.Question
+        Public Const vbRetryCancel As MsgBoxStyle = MsgBoxStyle.RetryCancel
+        Public Const vbSystemModal As MsgBoxStyle = MsgBoxStyle.SystemModal
+        Public Const vbYesNo As MsgBoxStyle = MsgBoxStyle.YesNo
+        Public Const vbYesNoCancel As MsgBoxStyle = MsgBoxStyle.YesNoCancel
```

Comment:
Please consider ordering constants to match reference source so it's clear where there are differences. Thanks.

Reply:
Done